src/textual/_xterm_parser.py

```diff
@@ -34,7 +39,7 @@ def __init__(
 
         super().__init__()
 
-    def debug_log(self, *args: Any) -> None:
+    def debug_log(self, *args: Any) -> None:  # pragma: no cover
```

Comment:
What was the typing error?

Reply:
There was no typing error. I was trying to ensure we had full test coverage of this module, and thought this code was temporary "for us" code so didn't think coverage.py should measure it. I can remove this and add a test case around it if you'd prefer.